pkg/kubelet/sysctl/whitelist.go

```diff
@@ -48,7 +48,7 @@ func NewWhitelist(patterns []string) (*patternWhitelist, error) {
 	}
```

Comment:
add unit tests to pkg/kubelet/sysctl/whitelist_test.go to make sure we can allow a sysctl in the format you want, and that the exact value and prefix matching work properly

Reply:
Ok, will update as soon as possible tomorrow.

Reply:
Updated.